2021-02-06 14:56:50,980 INFO    MainThread:24580 [wandb_setup.py:_flush():70] setting env: {}
2021-02-06 14:56:50,980 INFO    MainThread:24580 [wandb_setup.py:_flush():70] multiprocessing start_methods=fork,spawn,forkserver
2021-02-06 14:56:50,980 INFO    MainThread:24580 [wandb_setup.py:_flush():70] setting login settings: {}
2021-02-06 14:56:50,980 INFO    MainThread:24580 [wandb_setup.py:_flush():70] setting login settings: {}
2021-02-06 14:56:50,980 INFO    MainThread:24580 [wandb_init.py:_log_setup():318] Logging user logs to /home/iiitd/Desktop/Ahmad/doa2/wandb/run-20210206_145650-3h1klb24/logs/debug.log
2021-02-06 14:56:50,980 INFO    MainThread:24580 [wandb_init.py:_log_setup():319] Logging internal logs to /home/iiitd/Desktop/Ahmad/doa2/wandb/run-20210206_145650-3h1klb24/logs/debug-internal.log
2021-02-06 14:56:53,321 INFO    MainThread:24580 [wandb_run.py:_console_start():1308] atexit reg
2021-02-06 14:56:53,321 INFO    MainThread:24580 [wandb_run.py:_redirect():1175] redirect: SettingsConsole.REDIRECT
2021-02-06 14:56:53,321 INFO    MainThread:24580 [wandb_run.py:_redirect():1178] Redirecting console.
2021-02-06 14:56:53,321 INFO    MainThread:24580 [redirect.py:install():205] install start
2021-02-06 14:56:53,322 INFO    MainThread:24580 [redirect.py:install():220] install stop
2021-02-06 14:56:53,322 INFO    MainThread:24580 [redirect.py:install():205] install start
2021-02-06 14:56:53,322 INFO    MainThread:24580 [redirect.py:install():220] install stop
2021-02-06 14:56:53,322 INFO    MainThread:24580 [wandb_run.py:_redirect():1222] Redirects installed.
2021-02-06 14:59:25,836 INFO    MainThread:24580 [wandb_run.py:_atexit_cleanup():1278] got exitcode: 1
2021-02-06 14:59:25,836 INFO    MainThread:24580 [wandb_run.py:_restore():1250] restore
2021-02-06 14:59:25,836 INFO    MainThread:24580 [redirect.py:uninstall():224] uninstall start
2021-02-06 14:59:25,836 INFO    MainThread:24580 [redirect.py:_stop():279] _stop: stdout
2021-02-06 14:59:25,837 INFO    stdout    :24580 [redirect.py:_pipe_relay():121] relay done saw last write: stdout
2021-02-06 14:59:25,837 INFO    stdout    :24580 [redirect.py:_pipe_relay():137] relay done done: stdout
2021-02-06 14:59:25,837 INFO    MainThread:24580 [redirect.py:_stop():285] _stop closed: stdout
2021-02-06 14:59:25,837 INFO    MainThread:24580 [redirect.py:_stop():291] _stop joined: stdout
2021-02-06 14:59:25,837 INFO    MainThread:24580 [redirect.py:_stop():293] _stop rd closed: stdout
2021-02-06 14:59:25,837 INFO    MainThread:24580 [redirect.py:uninstall():228] uninstall done
2021-02-06 14:59:25,837 INFO    MainThread:24580 [redirect.py:uninstall():224] uninstall start
2021-02-06 14:59:25,837 INFO    MainThread:24580 [redirect.py:_stop():279] _stop: stderr
2021-02-06 14:59:25,837 INFO    stderr    :24580 [redirect.py:_pipe_relay():121] relay done saw last write: stderr
2021-02-06 14:59:25,837 INFO    stderr    :24580 [redirect.py:_pipe_relay():137] relay done done: stderr
2021-02-06 14:59:25,837 INFO    MainThread:24580 [redirect.py:_stop():285] _stop closed: stderr
2021-02-06 14:59:25,837 INFO    MainThread:24580 [redirect.py:_stop():291] _stop joined: stderr
2021-02-06 14:59:25,838 INFO    MainThread:24580 [redirect.py:_stop():293] _stop rd closed: stderr
2021-02-06 14:59:25,838 INFO    MainThread:24580 [redirect.py:uninstall():228] uninstall done
2021-02-06 14:59:26,163 INFO    MainThread:24580 [wandb_run.py:_wait_for_finish():1395] got exit ret: uuid: "5ba85c9956cb4db79904482bae2717f2"
response {
  poll_exit_response {
    file_counts {
      wandb_count: 1
    }
    pusher_stats {
      uploaded_bytes: 511
      total_bytes: 511
    }
  }
}

2021-02-06 14:59:28,169 INFO    MainThread:24580 [wandb_run.py:_wait_for_finish():1395] got exit ret: uuid: "47c37a15280748499c6617a66ffaa15b"
response {
  poll_exit_response {
    done: true
    exit_result {
    }
    file_counts {
      wandb_count: 5
    }
    pusher_stats {
      uploaded_bytes: 5263
      total_bytes: 5263
    }
  }
}

2021-02-06 14:59:29,421 INFO    MainThread:24580 [wandb_run.py:_show_files():1614] logging synced files
2021-02-06 14:59:29,423 INFO    MainThread:24580 [internal.py:handle_exit():139] Internal process exited
